### DISCUSSION TITLE
Can't publish iOS Firebase App Distribution

### DISCUSSION THREAD

**Discussion by adhityapaf** [codemagic-ci-cd/codemagic-docs#1968] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

This discussion is the same as https://github.com/orgs/codemagic-ci-cd/discussions/1339 but closed and unanswered.
1. Following steps from [docs](https://docs.codemagic.io/yaml-publishing/firebase-app-distribution/)
2. Run a new build
3. Build success but failed to publish to the Firebase App Distribution  

### Expected results

iOS App distributed to Firebase App Distribution

### Actual results

An error occurred when publishing: `Skip publishing to Firebase App Distribution: no matching artefacts found.`

### Build id (optional)

64d9a637bd6ac080be77915a

**Comment by kulakarla** [comment]
Hello @adhityapaf,


This error means that Codemagic cannot find the artefacts.
After the build, do you have an .ipa available to download from the build interface?

**Reply by adhityapaf** [reply]
Hi @kulakarla,

Nope, it is only xcarchive (because I added it on `artifacts`).

Here's my `artifacts` content:
```
    artifacts:
      - build/ios/ipa/*.ipa
      - /tmp/xcodebuild_logs/*.log
      - flutter_drive.log
      - build/ios/archive/Runner.xcarchive
```

**Reply by kulakarla** [reply, marked as answer]
Modify your flutter build script to be like this:
`flutter build ipa --release --export-options-plist=/Users/builder/export_options.plist`

**Reply by adhityapaf** [reply]
Thank you @kulakarla, it works! 😄
